WordPressKit/ActivityServiceRemote_ApiVersion1_0.swift

```diff
@@ -10,20 +10,34 @@
     /// - Parameters:
     ///     - siteID: The target site's ID.
     ///     - rewindID: The rewindID to restore to.
+    ///     - types: The types of items to restore.
     ///     - success: Closure to be executed on success
     ///     - failure: Closure to be executed on error.
     ///
     /// - Returns: A restoreID to check the status of the rewind request.
     ///
     @objc public func restoreSite(_ siteID: Int,
                                   rewindID: String,
+                                  types: RestoreTypes? = nil,
                                   success: @escaping (_ restoreID: String) -> Void,
                                   failure: @escaping (Error) -> Void) {
         let endpoint = "activity-log/\(siteID)/rewind/to/\(rewindID)"
         let path = self.path(forEndpoint: endpoint, withVersion: ._1_0)
-        
+        var parameters: [String: AnyObject] = [:]
+
+        if let types = types {
+            var restoreTypes: [String: AnyObject] = [:]
```

Comment:
What about adding a `toDictionary()` method to the `RestoreTypes` since we convert it to a dictionary a few times?